microservices/nris_api/backend/app/nris/scheduled_jobs/nris_jobs.py

```diff
@@ -31,19 +41,13 @@ def run_nightly_nris_etl():
             sched.app.logger.info('XML Import completed')
             # TODO: Insert update into status table
 
+            clean_nris_etl_data()
+            etl_nris_data()
+            sched.app.logger.info('NRIS ETL Completed!')
```

Comment:
i believe this will crash if we try to run this job via flask cli.. maybe add some safety around this? I tried to abstract it with a get_logger util, maybe use that?

Reply:
I don't think it should since the command explicitly runs with the sched.app context?

Reply:
You're right thought that we do need to abstract the logger out of the app context, lets create an item for that.

Reply:
This should be fine. I agree though, having the logger abstracted would be nice.